lib/src/protobuf/generated_message.dart

```diff
@@ -345,3 +345,10 @@ abstract class GeneratedMessage {
   /// For generated code only.
   void $_setInt64(int index, Int64 value) => _fieldSet._$set(index, value);
 }
+
+/// The package name of a protobuf message.
+class PackageName {
```

Comment:
There are existing .proto files that define a message called PackageName.
Make sure this works (e.g. by using a library prefix in generated code).

Reply:
Yes, I should solve https://github.com/dart-lang/dart-protoc-plugin/issues/47 first.

Reply:
Now that the library prefix change has been merged, is this ready to go?

Reply:
Now the corresponding protoc  change has landed, and a new version is published!

Reply:
Nice! Great work on this